content/en/security/application_security/code_security/_index.md

```diff
@@ -19,7 +19,7 @@ further_reading:
 
 Datadog Code Security identifies code-level vulnerabilities in your services and provides actionable insights and recommended fixes.
 
-For a list of supported services, see [Library Compatibility Requirements][5].
+For a list of supported services, see the [Library Compatibility Requirements][5].
```

Comment:
```suggestion
For a list of supported services, see the [Library Compatibility Requirements][5].
```

I think something is wrong with this page `/security/application_security/code_security/setup/compatibility/` 

It correctly only shows the 4 supported libraries:
JAVA | .NET | NODE.JS | PYTHON

But then when I click on one of those say, Java: it takes me to a page where all of the libraries are listed so It's kinda confusing, I would expect it to just take me to a page that only has those 4 libraries:

<img width="624" alt="image" src="https://github.com/user-attachments/assets/64ce8246-1ce2-407f-acbf-8480850348b9">

Reply:
I forgot to remove the other language pages and so Hugo was displaying them at the top.
I will commit a fix.

Reply:
fix committed.